docs/tutorial/using-grpc-plugins.mdx

```diff
@@ -0,0 +1,473 @@
+---
+title: "Building and deploying gRPC plugins"
+description: "Learn how to use gRPC plugins with Cosmo & Cosmo Router"
+icon: plug
+---
+
+## Prerequisites
+
+- [Cosmo CLI](/cli/intro) installed and configured
+
+## Overview
+
+This tutorial shows you how to create a gRPC plugin subgraph from scratch using the Cosmo CLI.
+
+You'll start by initializing a new plugin, and we'll walk you through implementation and deployment.
+
+Here's a short version of the steps for reference:
+
+<Steps>
+  <Step title="Initialize Plugin">
+    Create a new gRPC plugin using `wgc router plugin init <name>` and define your GraphQL schema
+    
+    [→ Jump to initialization](#initialize-plugin)
+  </Step>
+  <Step title="Generate & Implement">
+    Generate Protobuf code with `wgc router plugin generate` and implement your resolvers in Go
+    
+    [→ Jump to implementation](#generate-and-implement)
+  </Step>
+  <Step title="Publish & Deploy">
+    Publish your plugin to Cosmo with `wgc router plugin publish` and test via GraphQL queries
+    
+    [→ Jump to deployment](#publish-and-deploy)
+  </Step>
+</Steps>
+
+---
+
+First off, let's get familiar with some terminology we use to describe different parts of the platform:
+
+- **Cosmo Cloud**: Our cloud-hosted platform for publishing and monitoring your subgraphs
+- **Supergraph**: The unified graph containing the composition of all your subgraphs
+- **Subgraph**: A small part of a supergraph, usually serving a small service contract. For example, “billing”
+- **gRPC Subgraph**: A subgraph built with our new gRPC-based tooling that can be run standalone or as a plugin
+- **gRPC Plugin**: A way to run subgraphs created with gRPC alongside the router directly in your infrastructure, no extra deployment required
+
+Now that we have that sorted, let’s move onto the good stuff.
+
+## Initialize Plugin
+
+### Your first gRPC plugin
+
+gRPC plugins are the best way to get started with Cosmo Connect. They support all the same features as gRPC services, but you don’t have to create separate deployments, handle networking, or manage inter-service authentication.
```

Comment:
We haven't explained Connect. I'd not mention it. We don't have yet a clear path.